.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2007-present the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
At the moment I don't know anything about mixing licenses. Until now my project was GPL.

Reply:
Oh OK, I understand.
Maybe this helps: https://resources.whitesourcesoftware.com/blog-whitesource/top-10-apache-license-questions-answered#:~:text=Is%20the%20Apache%20License%20compatible,released%20under%20these%20two%20licenses.